demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -97,7 +158,7 @@ def formatted_error_str(error_type):
 
         formatted_error = formatted_error_str('ERROR')
         if should_print and not self.suppress_print:
-            if suggested_fix:
+            if suggested_fix and 'can not be ignored in .pack-ignore' not in formatted_error:
```

Comment:
Not urgent, but using strings for conditionals is not the best practice, can we use some other flag instead?

Reply:
you are right, ill handle it in a different way.